common/openapi.yml

```diff
@@ -260,6 +260,19 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/status/v1:
+    get:
+      summary: Check status of the service.
+      tags: [ public ]
+      operationId: getStatusDetail
+      security: [ ]
+      responses:
+        '200':
+          $ref: '#/components/responses/SystemStatusDetailResponse'
+        '500':
+          $ref: '#/components/responses/ServerError'
+        '503':
+          $ref: '#/components/responses/SystemStatusDetailResponse'
   /status:
```

Comment:
Should we mark the old `status` endpoint as deprecated?

Reply:
Not a bad idea!

Reply:
Done!